### PR TITLE
Scroll to selection

### PR DESCRIPTION
**Pull request by MSGhero**
Closes #1467, #1825

Fixes `scrollV`, `maxScrollV`, and `bottomScrollV` behavior. Also adds "scroll to selection" as you type in an input field, move the caret with the arrow keys, or use `setSelection()`.

It could be optimized I bet.

### CODE REVIEW

**Comment by MSGhero** [comment]
Another alternative is for LRUD text navigation to use `setSelection()` instead of manually setting the values. It would reduce the number of lines a bit, functionality would be the same.

**Comment by MSGhero** [comment]
Whoops... dev is targeted now

**Comment by MSGhero** [comment]
Alright this works now

File: src/openfl/text/TextField.hx [hunk 1]
```diff
@@ -51,84 +53,84 @@ import js.html.DivElement;
 #end
 
 /**
-	The TextField class is used to create display objects for text display and
```
Comment (Gama11) [inline src/openfl/text/TextField.hx]: Looks like there were some changes to the doc comment style you'd have to merge manually.

**Comment by MSGhero** [comment]
I've been running this for a month without issues.

**Pull request merged**